### PR TITLE
Fixed inspection-report issue, added a new api call to get the inspec…

### PR DESCRIPTION
**Pull request by BenHakimIlyass**
…tion pdf report but still WIP we get 500

## Technical description

- [x] Fixed inspection-report start issue, by downgrade expo version from `expo45.0.4` to `expo44.0.6`
- [x] Created a new api call to get the inspection pdf report
- [x] WIP still getting 500 error (even on postman)  
- [x] Fixed IRA theme issue
- [x] Added date to inspectionDetails screen title

## Screenshots (optional):

No screenshots.

*This Pull Request template has been written and generated by Monk JS repository.*

### CODE REVIEW

**Comment by natsukiai** [comment]
It's OK for the WIP PDF, it will help back-end to test in dev. Good job Ilyass.

File: services/inspection-report/src/views/InspectionDetails/ImageList/index.js [hunk 1]
```diff
@@ -59,11 +59,11 @@ export default function ImageList({ itemData }) {
...
-                options={{
-                  label: {
-                    fontSize: 12,
-                  },
-                }}
+                // options={{
+                //   label: {
+                //     fontSize: 12,
+                //   },
+                // }}
```
[the remark this answers is not in the document]
Comment (natsukiai) [reply services/inspection-report/src/views/InspectionDetails/ImageList/index.js:66]: Hey Ilyass, can you put more details on the error in the #293 issue please?

[the remark this answers is not in the document]
Comment (BenHakimIlyass) [reply services/inspection-report/src/views/InspectionDetails/ImageList/index.js:66]: On it

**Comment by natsukiai** [comment]
Not a big deal, but just some linting errors. We must had changed the eslint package version. ^^

Capture d’écran 2022-06-15 à 16 47 10

**Review by natsukiai — APPROVED** [review]

**Pull request merged**